lib/semantics/check-do.cc

```diff
@@ -27,13 +28,32 @@
 namespace Fortran::semantics {
 
 using namespace parser::literals;
+using namespace common;
```

Comment:
Please don't use `using namespace` except for namespaces specifically limited for user-defined literals.

Reply:
Thanks, Peter.  Won't do.